src/UICrafter.Core/Utility/ColorOptions.cs

```diff
@@ -0,0 +1,21 @@
+namespace UICrafter.Core.Utility;
+using UICrafter.Core.Models;
+using MudBlazor;
+
+
+public class ColorOptions
+{
+	public static Color GetColor(BaseModel Model)
+	{
+		return Model.Color?.ToLower() switch
```

Comment:
Dotnet has string compare options where you can set ignore case. So Idk about this toLower() there should be some structure where it just checks but ignores casing